samples/angular-todo/src/app/features/journey/confirmation/confirmation.component.html

```diff
@@ -0,0 +1,16 @@
+<!--
+angular-todo-prototype
+
+confirmation.component.html
+
+Copyright (c) 2021 ForgeRock. All rights reserved.
+This software may be modified and distributed under the terms
+of the MIT license. See the LICENSE file for details.
+-->
+<div class="cstm_form d-grid gap-2">
+  <ng-container *ngFor="let option of callback.getOptions()">
+    <button class="cstm_form btn btn-primary w-100" (click)="onSetOptionValue(option)">
+      {{ option }}
+    </button>
```

Comment:
Let's turn this into a select element or radio inputs like I mention in my Gist here: https://gist.github.com/cerebrl/766f7df333bcf6f180fa0ac8de4010d2. That way, we don't have to worry about figuring out if we need to render the submit button or not in this case.